### DISCUSSION TITLE
急！急！急！

### DISCUSSION THREAD

**Discussion by Teriri-Apocalypse** [share121/inter-knot#264] [其他]
![6ccfdf6f378b5f79b7eb698d315091bb](https://github.com/user-attachments/assets/0fff07c8-c843-41e6-907b-4480c9df08bc)

本主教那在罗浮仙舟当太卜的表姐近日来信说罗浮缺临时工，现发布委托招募胆大的英雄豪杰！
工作内容：
1.协助罗浮十王司和云骑军处理丰饶孽物（须直面呼雷等逃犯，冲锋在第一线）
2.协助仙舟联盟清除叛徒（可能要和持明打架，还是冲锋在第一线）
任职要求：
1.胆大
2.胆大
3.胆大
报酬：
1.你的小命
2.本主教表妹纳西妲的智慧

**Comment by Luckyxinchen** [comment]
一看就知道是黑心雇主😡

**Comment by changline** [comment]
这个委托我接了

**Comment by Elysia-Ego-093** [comment]
这个委托我不接，除非1600菲林(doge)
